pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -352,7 +352,12 @@ func makeFirewallName(name string) string {
 // TODO(a-robinson): Don't just ignore specified IP addresses. Check if they're
 // owned by the project and available to be used, and use them if they are.
 func (gce *GCECloud) EnsureTCPLoadBalancer(name, region string, externalIP net.IP, ports []*api.ServicePort, hosts []string, affinityType api.ServiceAffinity) (*api.LoadBalancerStatus, error) {
-	glog.V(2).Info("Checking if load balancer already exists: %s", name)
+	if len(hosts) == 0 {
+		glog.Errorf("Call to EnsureTCPLoadBalancer with no hosts")
```

Comment:
This log statement isn't really needed, the error you return will be logged at a higher level.